src/import-utils.ts

```diff
@@ -131,10 +144,17 @@ export async function inlineScript(document: Document, scriptTag: ASTNode) {
   }
 
   // Second argument 'true' tells encodeString to escape <script> tags.
-  const scriptContent =
-      encodeString(scriptImport.document.parsedDocument.contents, true);
+  let scriptContent = scriptImport.document.parsedDocument.contents;
+
+  if (enableSourcemaps) {
+    // It simplifies line calculation logic if inlined scripts don't
+    // start on the same line as the opening <script> tag
```

Comment:
This comment reads like you're adding a newline to the beginning of the source but I don't see one here. Looks like you fixed the issue with the initial line in the source. Is this comment still relevant?